challenge_take_out_the_garbage/src/challenge_take_out_the_garbage/pick_up.py

```diff
@@ -261,6 +286,9 @@ def __init__(self, robot, trashbin_designator, arm_designator):
         smach.StateMachine.__init__(self, outcomes=["succeeded", "failed", "aborted"])
         place_pose_designator = EmptySpotDesignator(robot, trashbin_designator, arm_designator)
 
+        if not place_pose_designator:
```

Comment:
This is not possible. Designator doesn't have a `__bool__`. This will just check if the object is not None or similar.